public/app/features/alerting/unified/AlertEditor.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { FC } from 'react';
+import { PageToolbar, ToolbarButton, stylesFactory, Form } from '@grafana/ui';
+
+import { GrafanaTheme, SelectableValue } from '@grafana/data';
+import { css } from 'emotion';
+import { config } from 'app/core/config';
+import AlertTypeSection from './components/AlertTypeSection';
+import AlertConditionsSection from './components/AlertConditionsSection';
+import Expression from './components/Expression';
+
+import { fetchRulerRulesNamespace, setRulerRuleGroup } from './api/ruler';
+import { RulerRuleDTO } from 'app/types/unified-alerting/dto';
+
+type Props = {};
+
+interface AlertRuleDTO {
```

Comment:
naming niggle: DTO stands for "Data Transfer Object", that is, sturcture that is sent over network. This is problably should be `AlertRuleFormValues` or smth like that